### PR TITLE
Edits to text in Delete By Query API doc

### PR DESCRIPTION
**Pull request by dmeiss**
Minor edits to text for punctuation, capitalization, and word usage. 

Throughout the doc I made “delete by query” into “ `_delete_by_query`” for consistency and for readability.

### CODE REVIEW

**Comment by danielmitterdorfer** [comment]
@elasticmachine test this please

**Review by danielmitterdorfer — COMMENTED** [review]
Overall the changes look fine to me. I also think it's good that we refer to the delete by query API consistently now but I am not certain whether we should refer to it with the API path part (i.e. `_delete_by_query`). Maybe just calling it "delete by query API" would be easier to read (I am not a native speaker though)?

**Comment by dmeiss** [comment]
I wasn't certain we should refer to it that way either, to be honest. The doc uses "Delete By Query", "delete-by-query", "delete by query", and "_delete_by_query" throughout the text, so I'm not sure there's a perfect solution. I'm thinking maybe a good solution is to undo my changes, and just change the instances of "Delete By Query" and "delete-by-query" to simply "delete by query" (i.e., make lowercase & remove hyphens), which will make it seem much more consistent. What do you think?

**Comment by danielmitterdorfer** [comment]
> [...] change the instances of "Delete By Query" and "delete-by-query" to simply "delete by query" (i.e., make lowercase & remove hyphens), which will make it seem much more consistent.

That makes sense to me.

**Comment by dmeiss** [comment]
Changed the "_delete_by_query" references back to "delete by query". Thanks for the input!

File: docs/reference/docs/delete-by-query.asciidoc [hunk 1]
```diff
@@ -163,10 +163,10 @@ for details. `timeout` controls how long each write request waits for unavailabl
 shards to become available. Both work exactly how they work in the
 <<docs-bulk,Bulk API>>. As `_delete_by_query` uses scroll search, you can also specify
 the `scroll` parameter to control how long it keeps the "search context" alive,
-eg `?scroll=10m`, by default it's 5 minutes.
+e.g. `?scroll=10m`. By default it's 5 minutes.
 
 `requests_per_second` can be set to any positive decimal number (`1.4`, `6`,
-`1000`, etc) and throttles rate at which `_delete_by_query` issues batches of
+`1000`, etc.) and throttles the rate at which `_delete_by_query` issues batches of
```
Comment (danielmitterdorfer) [inline docs/reference/docs/delete-by-query.asciidoc]: I think this instance of `_delete_by_query` is an oversight?

File: docs/reference/docs/delete-by-query.asciidoc [hunk 2]
```diff
@@ -570,7 +570,7 @@ are distributed proportionally to each sub-request. Combine that with the point
 above about distribution being uneven and you should conclude that the using
 `size` with `slices` might not result in exactly `size` documents being
 `_delete_by_query`ed.
```
Comment (danielmitterdorfer) [inline docs/reference/docs/delete-by-query.asciidoc]: I think we can just refer to this one as "deleted"?

**Review by danielmitterdorfer — COMMENTED** [review]
I found two more instances of `_delete_by_query` but apart from that it looks good.

**Comment by dmeiss** [comment]
Made both changes.

**Review by danielmitterdorfer — APPROVED** [review]
Thanks for iterating. The changes look good to me. I'll trigger CI once more before merging it.

**Comment by danielmitterdorfer** [comment]
@elasticmachine test this please

**Pull request merged**